uiowa/uiowa
# Feature: uids_base events/footer/safari fixes

In this PR:

Update to uids_base theme to address:
- https://github.com/uiowa/uids/issues/93

UIDS bump that includes fixes for:
- https://github.com/uiowa/uids/issues/95
- https://github.com/uiowa/uids/issues/91
- https://github.com/uiowa/uids/issues/90
- https://github.com/uiowa/uids/issues/89

# How to test

1. Safari scroll issues

Go to https://uiowa.local.drupal.uiowa.edu in Safari and scroll up and down in multiple spots.  There use to be a 40 pixel gap occurring in the right side of the scrollbar and this gap should be gone now. 

2. Footer Updates

- Verify that footer icon no longer appears next to phone number.   
- Verify that there is an updated footer socket menu (see screenshot below)

<img width="444" alt="Screen Shot 2020-04-07 at 12 56 18 PM" src="https://user-images.githubusercontent.com/1036433/78703090-3d495180-78cf-11ea-98a9-2bc6440f842c.png">

3. Event Updates

- Verify that Events background pattern on https://uiowa.local.drupal.uiowa.edu is a white brain pattern after clearing your browser cache. 

4.  Card refactor

Edit the cards on https://uiowa.local.drupal.uiowa.edu and verify that you can:

- Verify that the link buttons on the cards are rendering with a faux link button and that entire card is clickable
- Verify that `aria-describedby` is added to heading link `a` and `aria-hidden=true` is added to faux button
- Remove the photo and save the block and verify that the block renders without an image or image markup
- Remove the link text and save the block and have the block render without a faux link button
- Remove the title and save the block and have the block render without a title
- Remove link url and save the block and have the block render without a link
- Remove title while having a link url and verify that entire block is clickable
